kfs-core/src/main/resources/edu/arizona/kfs/gl/businessobject/datadictionary/DocumentHeaderLiteBo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+    <bean id="DocumentHeaderLiteBo" parent="BusinessObjectEntry">
+        <property name="businessObjectClass" value="edu.arizona.kfs.gl.businessobject.lite.DocumentHeaderLiteBo" />
+        <property name="objectLabel" value="DocumentHeaderLiteBo" />
+        <property name="attributes">
+            <list>
+                <ref bean="DocumentHeaderLiteBo-documentId" />
+                <ref bean="DocumentHeaderLiteBo-documentTypeId" />
+                <ref bean="DocumentHeaderLiteBo-docRouteStatus" />
+                <ref bean="DocumentHeaderLiteBo-docRouteLevel" />
+                <ref bean="DocumentHeaderLiteBo-appDocStatus" />
+                <ref bean="DocumentHeaderLiteBo-dateModified" />
+                <ref bean="DocumentHeaderLiteBo-createDate" />
+                <ref bean="DocumentHeaderLiteBo-approvedDate" />
+                <ref bean="DocumentHeaderLiteBo-finalizedDate" />
+                <ref bean="DocumentHeaderLiteBo-routeStatusDate" />
+                <ref bean="DocumentHeaderLiteBo-appDocStatusDate" />
+                <ref bean="DocumentHeaderLiteBo-docTitle" />
+                <ref bean="DocumentHeaderLiteBo-appDocId" />
+                <ref bean="DocumentHeaderLiteBo-docVersion" />
+                <ref bean="DocumentHeaderLiteBo-initiatorWorkflowId" />
+                <ref bean="DocumentHeaderLiteBo-routedByUserWorkflowId" />
+                <ref bean="DocumentHeaderLiteBo-objectId" />
+                <ref bean="DocumentHeaderLiteBo-versionNumber" />
+            </list>
+        </property>
+    </bean>
+
+<!--     Attribute Definitions -->
+    <bean id="DocumentHeaderLiteBo-documentId"              parent="DocumentRouteHeaderValue-documentId"                                            />

```

Comment:
Same here w/ the formatting. I see what the intent is, but the counterargument is that now I'm force to scroll all the way to the right to verify that it is indeed the end of the element, when that shouldn't be necessary,